compiler/record-minmax/CMakeLists.txt

```diff
@@ -23,6 +23,27 @@ if(NOT ENABLE_TEST)
   return()
 endif(NOT ENABLE_TEST)
 
+# create record-minmax-for-thread-test target
+# Note: record-minmax-for-thread-test is built with -fsanitize=thread so that thread sanitizer can check memory bugs,
+# record-minmax is built without the option for performance.
+add_executable(record-minmax-for-thread-test ${DRIVER} ${SOURCES})
+target_include_directories(record-minmax-for-thread-test PRIVATE include)
+
+target_link_libraries(record-minmax-for-thread-test arser)
+target_link_libraries(record-minmax-for-thread-test safemain)
+target_link_libraries(record-minmax-for-thread-test luci_import)
+target_link_libraries(record-minmax-for-thread-test luci_env)
+target_link_libraries(record-minmax-for-thread-test luci_export)
+target_link_libraries(record-minmax-for-thread-test luci_interpreter)
+target_link_libraries(record-minmax-for-thread-test dio_hdf5)
+target_link_libraries(record-minmax-for-thread-test vconone)
+target_link_libraries(record-minmax-for-thread-test nncc_coverage)
+target_link_libraries(record-minmax-for-thread-test luci_log)
+
+install(TARGETS record-minmax-for-thread-test DESTINATION bin)
```

Comment:
This line still exist. @jinevening , is this OK?

Reply:
@BalyshevArtem Please remove the line.

Reply:
Sorry, forgot do rebase, thank you!